.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request
+description: Suggest an idea for the project
+labels:
+  - "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing feature requests before submitting your own."
+  - type: textarea
+    id: enhancement-summary
+    attributes:
+      label: Enhancement Summary
+      description: |
+        - Briefly describe the enhancement.
+        - Explain how the proposed enhancement helps overcome a limitation of LMMS you are currently facing.
+    validations:
+      required: true
+  - type: textarea
+    id: justification
+    attributes:
+      label: Justification
+      description: |
+        - Describe why you believe this to be a good idea, and how you think it would improve LMMS.
+        - Is there a reason why this should be included in LMMS itself and not as a plugin?
```

Comment:
We don't have a policy of not implementing things that are available as third-party plugins (except perhaps instruments that include big sample libraries), so I'm not sure if this point is necessary.